config/sections/mixins/empty.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+use Kirby\Toolkit\I18n;
```

Comment:
It works without this line because of other updates

Reply:
Awesome! Thanks for verifying. 👍